src/pages/Reviews.js

```diff
@@ -160,8 +168,8 @@ const Reviews = ({ productId }) => {
           value={formData.comment}
           onChange={handleInputChange}
         /></div>
-        
-        <button onClick={createReview}>Create</button>
```

Comment:
If you do `onClick={createReview}` then React is going to pass the whole `event.target.value`  to the function `createReveiw`. Here, that meant it was passing the full review form fields in. But that's not what `createReview` is expecting, it's expecting to just get `productId`, and then it fetches the field data itself using React state.

So update this to instead use the anonymous function notation `() => ...` and then call the `createReview` function with the appropriate arguments

Reply:
@akosasante didn't know that, thak you for explaining!!